resolver/ng/defaults.yaml

```diff
@@ -0,0 +1,26 @@
+Defaults:
+  ng:
+    resolvconf:
+      enabled: False
+      remove: True
+      file: /run/resolvconf/resolv.conf
+  nameservers:
+    - 8.8.8.8
+    - 8.8.4.4
+  domain: ''
+  searchpaths: []
+  options: []
+
+# os_family vars.
```

Comment:
I understand it is best practice to set primary variables in defaults.yaml and pillar.example, and defer explicit os handling to  'filter_by' block in map.jinja.  Therefore pillar.example and defaults.yaml would not mention 'Debian' or 'Ubuntu' but simply the following ...  could you consider this?

```
  ng:
 +    resolvconf:
 +      enabled: True
 +      remove: False
 +      file: /etc/resolvconf/run/resolv.conf
```